packages/pie/src/PieCanvas.tsx

```diff
@@ -275,10 +293,10 @@ const PieCanvas = ({
 
     const { showTooltipFromEvent, hideTooltip } = useTooltip()
 
-    const handleMouseHover = event => {
+    const handleMouseHover = (event: React.MouseEvent<HTMLCanvasElement>) => {
         const datum = getArcFromMouse(event)
         if (datum) {
-            onMouseMove && onMouseMove(datum, event)
+            if (onMouseMove) onMouseMove(datum, event)
```

Comment:
```suggestion
            onMouseMove?.(datum, event)
```

Cleaner approach. Tested locally and prettier, lint, build all still work, so we should use the new syntax!

Reply:
You're right, thanks! updated.